compiler/src/iree/compiler/Codegen/Utils/GPUUtils.h

```diff
@@ -185,11 +185,14 @@ FailureOr<ArrayAttr> getSupportedMmaTypes(mlir::FunctionOpInterface entryPoint);
 /// Returns null TargetAttr othersise.
 IREE::GPU::TargetAttr getCLGPUTarget(MLIRContext *context);
 
+/// Returns the GPU target attribute from attribute |attr| if found.
+/// Returns null TargetAttr otherwise.
+IREE::GPU::TargetAttr getGPUTargetAttr(Attribute attr);
 /// Returns the GPU target attribute from executable |target| if found.
-/// Returns null TargetAttr othersise.
+/// Returns null TargetAttr otherwise.
 IREE::GPU::TargetAttr getGPUTargetAttr(IREE::HAL::ExecutableTargetAttr target);
```

Comment:
I think we can collapse the logic to the other function, and we delete this function, right? See the example:

https://github.com/iree-org/iree/blob/c8ba691b42354957abc8130d750cf16cbf0ec0bf/compiler/src/iree/compiler/Codegen/Utils/Utils.cpp#L67-L87

Reply:
Yeah, makes sense, I combined to logic and removed that function.